plugin/storage/integration/grpc_test.go

```diff
@@ -44,6 +44,8 @@ func (s *GRPCStorageIntegrationTestSuite) initialize() error {
 	err := command.ParseFlags([]string{
 		"--grpc-storage-plugin.binary",
 		s.pluginBinaryPath,
+		"--grpc-storage-plugin.log-level",
```

Comment:
I think I added this when I was trying to determine why the plugin wasn't working. It's not necessary but I don't think it hurts anything either.